crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -41,11 +41,11 @@ use std::{
 ///
 /// [`Or`]: crate::query::Or
 pub trait WorldQuery {
-    type Fetch: for<'a> Fetch<'a, State = Self::State>;
+    type Fetch: for<'w, 's> Fetch<'w, 's, State = Self::State>;
     type State: FetchState;
 }
 
-pub trait Fetch<'w>: Sized {
+pub trait Fetch<'w, 's>: Sized {
```

Comment:
I like the w/s shorthand elsewhere, but can we make these lifetimes `` `world`` and `` 'state`` to make them self-documenting??

Reply:
done